package.json

```diff
@@ -14,6 +14,7 @@
 		"stylelint:fix": "stylelint css/*.css css/*.scss src/**/*.scss src/**/*.vue --fix"
 	},
 	"dependencies": {
+		"@juliushaertl/vue-richtext": "^1.0.1",
```

Comment:
Any reason for not moving this under the Nextcloud org?

Reply:
Feel free to poke here https://github.com/juliushaertl/vue-richtext/issues :D